cmd/aro/main.go

```diff
@@ -44,11 +44,6 @@ func main() {
 	audit := utillog.GetAuditEntry()
 	log := utillog.GetLogger()
 
-	// TODO: Use `azuretypes.Platform.IsARO()` from github.com/openshift/installer/pkg/types/azure
-	if !platformIsAro {
```

Comment:
I don't have a good understanding of why we're removing this tag. It looks like it was originally used to disable managed identity. Do we disable managed identity somewhere else now?

Reply:
This tag controls code in the vendored Installer. Since that code no longer exists, we can remove this. It remains in openshift/ARO-Installer.